profile/README.md

```diff
@@ -1,33 +1,13 @@
 [skonfig](https://skonfig.li) is a system configuration and automation framework
 designed to work with all systems, from your toaster to the data centre.
 All you need is an SSH connection and a POSIX/UNIX-like shell environment.
-
 And best of all: you already know how to use it (if you know shell scripts).
 
-skonfig uses three main repositories:
+skonfig has three main repositories:
 
 * [skonfig](https://github.com/skonfig/skonfig) - implementation of the **skonfig** executable,
 * [base](https://github.com/skonfig/base) - explorer and types for **general use**,
 * [extra](https://github.com/skonfig/extra) - **special purpose** types and incubator for new types.
 
 Check out the [skonfig](https://github.com/skonfig/skonfig) repository to learn
 how to get started.
-
-**Need support?** You can find us in `#skonfig:matrix.org` ([matrix?](https://matrix.org/faq/)).
```

Comment:
I would keep the `Need help?` section here. When users reach our GH they can immediately see the channel.